stdlib/2and3/logging/__init__.pyi

```diff
@@ -226,6 +226,8 @@ class LogRecord:
 
 
 class LoggerAdapter:
+    logger = ...  # type: Logger
+    extra = ...  # type: Mapping[str, Any]
```

Comment:
For new code we prefer PEP 526 variable annotation syntax, e.g.`extra: Mapping[str, Any]`.

Reply:
Fixed.